Mathlib/Algebra/Category/ModuleCat/Presheaf/Sheafify.lean

```diff
@@ -368,7 +368,7 @@ variable {M₀' : PresheafOfModules.{v} R₀} {A' : Sheaf J AddCommGrp.{v}}
 induced by morphisms `τ₀ : M₀ ⟶ M₀'` and `τ : A ⟶ A'`
 which satisfy `τ₀.hom ≫ φ' = φ ≫ τ.val`. -/
 @[simps]
-def sheafifyMap : sheafify α φ ⟶ sheafify α φ' where
+def sheafifyMap (fac : τ₀.hom ≫ φ' = φ ≫ τ.val) : sheafify α φ ⟶ sheafify α φ' where
```

Comment:
Should `fac` in the variable block above move down?

Reply:
In fact, just removed.